sdcm/cluster_aws.py

```diff
@@ -242,15 +240,16 @@ def __init__(self, ec2_instance, ec2_service, credentials,
                                       base_logdir=base_logdir,
                                       node_prefix=node_prefix,
                                       dc_idx=dc_idx)
+        tags_list = [{'Key': 'Name', 'Value': name},
+                     {'Key': 'workspace', 'Value': cluster.WORKSPACE},
+                     {'Key': 'uname', 'Value': ' | '.join(os.uname())}]
         if cluster.TEST_DURATION >= 24 * 60:
             self.log.info('Test duration set to %s. '
                           'Tagging node with {"keep": "alive"}',
                           cluster.TEST_DURATION)
-            self._ec2_service.create_tags(Resources=[self._instance.id],
-                                          Tags=[{'Key': 'keep', 'Value': 'alive'}])
+            tags_list += [{'Key': 'keep', 'Value': 'alive'}]
```

Comment:
IMHO append to list is better then two lists concatenation:
tags_list.append({'Key': 'keep', 'Value': 'alive'})